docs/javadoc/allclasses-frame.html

```diff
@@ -3,7 +3,7 @@
 <html lang="de">
 <head>
 <!-- Generated by javadoc -->
-<title>All Classes (PowerSystemDataModel 1.1-SNAPSHOT API)</title>
+<title>All Classes (PowerSystemDataModel 2.0.0 API)</title>
```

Comment:
This is invalid, as this is actually a SNAPSHOT. We might have to modify the finalize task here. Pls consult me for further discussion.

Reply:
The longer I think about this, the more I would prefer just keeping it that way and don't publish javadocs for snapshot builds. Hence we can keep the versioning for now but we might consider removing the java doc generation from the gradle `finalizePR` task (and adapt the CONTRIBUTING.MD accordingly). Furthermore we could discuss about having one special branch for javadocs only and automatically deploy new javadocs on each release (compare https://vaadin.com/blog/host-your-javadoc-s-online-in-github) via jenkins. This of course would not produce snapshot javadocs anymore but I think we don't need them at all. In addition this would not allow us to review javadocs anymore but tbh I would assume none of us ever has reviewed javadocs on detail (especiall not if they are 4k+ of lines ...)